markdown/setup/dbeaver-driver-setup.md

```diff
@@ -0,0 +1,52 @@
+### DBeaver 
+[Link to product webpage](https://dbeaver.io/download/).
+
+#### Adding the Amazon Timestream JDBC Driver
+1. [Download](https://github.com/awslabs/amazon-timestream-driver-jdbc/releases/latest) the Timestream JDBC driver shaded JAR file (e.g., `amazon-timestream-jdbc-2.0.0-shaded.jar`)
+
+2. Launch the DBeaver application.
+
+3. In the main menu, navigate to and select **Database > Driver Manager > New**.
+
+    a. In **Settings** tab, for **Driver Name** field, enter a descriptive name (e.g. `Timestream`)
+
+    b. In **Settings** tab, for **URL Template** field, enter your JDBC connection string. For example:
+      
+        ```
+        -- replace the values in "<>" with your own value
+        jdbc:timestream://Region=<region>;
+        ```
+
+        If your database requires an username and password, it can be entered in the connection string when setting up the driver
+
+        ```
+        -- replace the values in "<>" with your own value
+        jdbc:timestream://AccessKeyId=<myAccessKeyId>;SecretAccessKey=<mySecretAccessKey>;SessionToken=<mySessionToken>;Region=<myRegion>
+        ```
+
+        or username and password can be entered later when connecting to Amazon Timestream using DBeaver.
+
+        ![Example](../images/dbeaver/dbeaver1.png)
+
```

Comment:
The rendering is a little strange, I think it might be due to too many spaces on the left/beginning of the sentences. Can it be fixed?
![image](https://user-images.githubusercontent.com/96995091/207129804-7c7141cf-58cc-488f-a1fa-1791181fee5b.png)

Reply:
Fixed